neo/app/controllers/Application.scala

```diff
@@ -59,7 +60,8 @@ class Application @Inject()(val wsClient: WSClient,
   }
 
   def ws = WebSocket.accept[JsValue, JsValue] { request =>
-    ActorFlow.actorRef(out => NeoActor.props(out, wsClient, request.host, config))
+    val prop = BerryClient.props(new JSONParser(), manager, new QueryPlanner(), config)
+    ActorFlow.actorRef(out => RequestRouter.props(out, prop, config))
```

Comment:
correct. since `berry` knows how to ship the result, it can directly send to out. 
otherwise, the `RequestRouter` has to receive the result and redirect it to `out` which is not reflect in the code. AND, it's not necessary.